### PR TITLE
[MXNET-250] CSS and .htaccess fix

### PR DESCRIPTION
**Pull request by thinksanky**
## Description ##
[ ] Copy .htaccess from master to root folder.
[ ] Copy CSS fix (mxnet.css) into all the versions.
[ ] Test out link - http://34.207.78.24/

### CODE REVIEW

**Comment by aaronmarkham** [comment]
This is fine for now. Package links can be re-enabled later.

File: docs/build_version_doc/build_all_version.sh [hunk 1]
```diff
@@ -68,6 +68,7 @@ for tag in $tag_list; do
           git checkout master -- docs/mxdoc.py
     fi
     git submodule update || exit 1
+    git submodule update --init --recursive
```
Comment (eric-haibin-lin) [inline docs/build_version_doc/build_all_version.sh]: Why not update line 70?

Comment (thinksanky) [reply docs/build_version_doc/build_all_version.sh]: I have tried updating it together, but its unstable. I have found this to be better and working consistently

File: docs/build_version_doc/update_all_version.sh [hunk 2]
```diff
@@ -68,31 +73,51 @@ fi
...
 done
 
+function update_mxnet_css {
+  tag=$1
+  echo "Begin update fixes.."
+  # All fixes are done on the master branch of mxnet-incubator repository
+  # During a nightly build, these fixes will be patched to all the versions in the asf-site repository including the master folder under versions directory.
+  # copy <master folder location> <version folder location>
+  
+  #SIM: https://sim.amazon.com/issues/AWSDBUX-19727
```
[the remark this answers is not in the document]
Comment (thinksanky) [reply docs/build_version_doc/update_all_version.sh]: Done

File: docs/build_version_doc/update_all_version.sh [hunk 3]
```diff
@@ -68,31 +73,51 @@ fi
...
     # This Python script is expecting the tag_list.txt and it will use that as the entries to populate
+
     python AddVersion.py --root_url "$root_url" --file_path "$built/versions/$tag" --current_version "$tag" || exit 1
 
-    if [ $tag != 'master' ]
-    then 
-        python AddPackageLink.py --file_path "$built/versions/master/install/index.html" \
-                                                   --current_version "$tag" || exit 1
-    fi
+    #if [ $tag != 'master' ]
```
[the remark this answers is not in the document]
Comment (thinksanky) [reply docs/build_version_doc/update_all_version.sh]: Done

File: docs/build_version_doc/update_all_version.sh [hunk 4]
```diff
@@ -68,31 +73,44 @@ fi
...
+
 # Update the specified tags with the Versions dropdown
 for tag in $tag_list; do
     # This Python script is expecting the tag_list.txt and it will use that as the entries to populate
+
     python AddVersion.py --root_url "$root_url" --file_path "$built/versions/$tag" --current_version "$tag" || exit 1
 
-    if [ $tag != 'master' ]
-    then 
-        python AddPackageLink.py --file_path "$built/versions/master/install/index.html" \
```
Comment (aaronmarkham) [inline docs/build_version_doc/update_all_version.sh]: Why are these removed entirely?
